packages/babel-plugin/src/css-prop/__tests__/string-literal.test.tsx

```diff
@@ -364,12 +351,8 @@ describe('css prop string literal', () => {
         </div>
       `);
 
-    // TODO: Correct the hash mock so variables have a unique name instead of hash-test.
-    // expect(actual).toInclude('style={{"--var-hash-test":(x||"")+"px","--var-hash-test":y}}');
-    expect(actual).toInclude('style={{"--var-hash-test":(x||"")+"px"}}');
-    expect(actual).toInclude(
-      '.cc-hash-test{transform:translate3d(var(--var-hash-test),var(--var-hash-test),0);color:red}'
```

Comment:
Did we miss assertion for `color:red` 🤔?

Reply:
oops let me add it back in